app/src/main/res/values-bg/strings.xml

```diff
@@ -0,0 +1,746 @@
+    <string name="OK">"ОК"</string>
+    <string name="dialog_streaks_title">"Нова функция: Линии!"</string>
+    <string name="dialog_streaks_message">"Вашата база данни ще бъде обновена, за да поддържа новата функция „линии“."</string>
+    <string name="dialog_no_email_apps_title">"Няма намерени имейл приложения"</string>
+    <string name="dialog_no_email_apps_message">"Архивирането на данни изисква да имате инсталирано приложение за имейл, за да можете да изпратите csv файл по пощата. Моля инсталирайте имейл клиент и опитайте отново."</string>
+    <string name="debug_generate_random_data">"Генериране на случайни данни"</string>
+    <string name="debug_generate_full_data">"Генериране на пълна информация"</string>
+    <string name="debug_generate_data_message">"Всички съществуващи данни ще бъдат заменени. Желаете ли да продължите?"</string>
+    <string name="task_generating_random_data">"Генериране на случайни данни"</string>
+    <string name="task_loading_servings_history_title">"Зареждане на историята на дозите"</string>
+    <string name="time_scale">"Времева скала"</string>
+    <string name="history_to_generate">"Генериране на история"</string>
+    <string name="date_servings_star" translatable="false">"{fa-star 24dp @color/gold}"</string>
+    <string name="date_servings_star_large" translatable="false">"{fa-star 64dp @color/gold}"</string>
+    <string name="welcome_to_my_daily_dozen">"Добре дошли в моя Daily Dozen!"</string>
+    <string name="activity_welcome_text">"Използвайте това приложение на дневна база, за да следите храните, които аз препоръчвам за оптимално здраве и дълголетие в книгата си „Как да не умрем“."</string>
+    <string name="dialog_ask_user_to_rate_app_title">"Харесвате ли това приложение?"</string>
+    <string name="dialog_ask_user_to_rate_app_message">"Моля, дайте ни обратна връзка в Google Play Store."</string>
+    <string name="rate_now">"Изпратете обратна информация"</string>
+    <string name="not_now">"Пропусни"</string>
+    <string formatted="false" name="format_version">"версия %s"</string>
+    <string formatted="false" name="format_num_days">"%s дни"</string>
+    <string name="daily_reminder_title">"Напомняне за Daily Dozen"</string>
+    <string name="daily_reminder_text">"Актуализиране на вашите порции за днес"</string>
+    <string name="daily_reminder_settings">"Настройки за напомняне"</string>
+    <string name="enable_daily_reminder">"Активиране на напомнянето"</string>
+    <string name="remind_me_at">"Напомни ми на"</string>
+    <string name="vibrate">"Вибрация"</string>
+    <string name="play_sound">"Възпроизвеждане на звук"</string>
+    <string name="videos">"Видео клипове"</string>
+
+    <!--
+    NOTE: These strings are used to created the url to take the user to the overall food videos page. These strings
+    must not be translated.
+    -->
+    <string name="food_info_videos_beans" translatable="false">"бобови растения"</string>
```

Comment:
This and the rest of the food_info_videos_* strings cannot be translated from English. Please revert all of these.